docs/How-To-Guides/how-to-create-play.md

```diff
@@ -107,6 +107,8 @@ npx create-react-play@latest -c <the_play_id>
 
 - Continue developing your play. Happy coding.
 
+> **Note:** When creating different component files for your play _(A separate file to create a component that includes JSX elements, like a custom button component for example)_, please use either a `.jsx`, `.ts` or `.tsx` file extension. This is due to our platform now using Vite for building.
```

Comment:
Would `.ts` be fine? As far as I understand, when we use React components with Vite, the file extension should either be `.jsx` or `.tsx` right?

Reply:
we can use `.js` & `.ts` but we should use `.jsx` & `.tsx`